truthy-falsy.js

```diff
@@ -0,0 +1,216 @@
+// 1
+function setAlarm(isEmployed, isOnVacation){
+  return isEmployed && !isOnVacation;
+}
+console.log(setAlarm(true, false));
+
+// 2
+function closeCompare(firstNumber, secondNumber, numbersMargin = 0){
+  const numbersDiff = Math.abs(firstNumber - secondNumber);
+  if (numbersDiff <= numbersMargin) {
+    return 0;
+  }
+  if (firstNumber < secondNumber) {
+    return -1;
+  }
+  return 1;
+}
+
+// 3
+function or(firstBoolean, secondBoolean) {
+  if (firstBoolean === true) {
+    return true;
+  }
+  if (secondBoolean === true) {
+    return true;
+  }
+  return false;
+}
+console.log(or(true, false));
+
+function xor(xorFirstCondition, xorSecondCondition) {
+  if (xorFirstCondition && !xorSecondCondition) {
+    return true;
+  }
+  if (xorSecondCondition && !xorFirstCondition) {
+    return true;
+  }
+  return false;
+}
+console.log(xor(true, true));
+
+// 4
+function isTruthy(booleanVariable, onTruthy, onFalsy) {
```

Comment:
use `callbackOnTruthy` and `callbackOnFalsy` since typically, a function passed into another function is called a callback